core/modules/user/user.theme.inc

```diff
@@ -125,7 +130,11 @@ function theme_user_admin_permissions($variables) {
   }
   $header[] = (t('Permission'));
   foreach (element_children($form['role_names']) as $role_name) {
-    $header[] = array('data' => backdrop_render($form['role_names'][$role_name]), 'class' => array('checkbox'));
+    $header[] = array(
+      'data' => backdrop_render($form['role_names'][$role_name]),
+      'title' => check_plain(strip_tags($form['role_names'][$role_name]['#description'])),
```

Comment:
While testing on the PR sandbox, this line here throws this:

```
Notice: Undefined index: #description in theme_user_admin_permissions() (line 135 of /var/lib/tugboat/core/modules/user/user.theme.inc).
```

Got that when tried to edit a content type.

Reply:
Thanks, fixed in the new commits!